client/my-sites/plans-features-main/index.jsx

```diff
@@ -214,11 +216,13 @@ PlansFeaturesMain.defaultProps = {
 	isChatAvailable: false,
 	showFAQ: true,
 	site: {},
+	siteSlug: '',
 };
 
 export default connect(
-	state => ( {
+	( state, { site } ) => ( {
```

Comment:
We can save this for a future refactor, but looking at usages, I don't think it'd be too bad to also use selectedSite pulled directly from global state in connect.

Reply:
Yep! On my roadmap :+1: